### PR TITLE
My 2 cents

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

**Comment by damienalexandre** [comment]
Do you wish to rebase this PR? Some CS modifications have been done on master.

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
 php:
-  - 5.6
-  - 5.5
-  - 5.4
-  - 5.3
-  - hhvm
-  - 7
+  - 7.0
+  - 7.1
+  - 7.2
```
Comment (pounard) [inline .travis.yml]: You should add PHP 7.3 now

**Comment by lyrixx** [comment]
Here we go :tada: 

I wanted to drop support for psr-0. I was not able to do so. I don't know why :( and I gave up

**Comment by damienalexandre** [comment]
Tests are broken for 7.3 :sob: PHP CS Fixer is too old.

**Comment by lyrixx** [comment]
@damienalexandre Could we merge this one?

**Comment by damienalexandre** [comment]
Thanks a lot :+1:

**Pull request merged**